docs/_sass/_main.scss

```diff
@@ -1125,6 +1125,18 @@ button {
     line-height: 16px;
 }
 
+#platform-tabs > .badge{
```

Comment:
Can we just make this a more generic class? Like even just a simple `.badge` with no specificity before it.

Reply:
Also, why aren't we setting a font-size on these badges? Please see Danny's guidance.